### PR TITLE
Update units-and-global-variables.rst (Typo)

### PR DESCRIPTION
**Pull request by FedericoCapello**
Simple typo. Enforce consistency of explanation.

### CODE REVIEW

File: docs/units-and-global-variables.rst [hunk 1]
```diff
@@ -22,8 +22,8 @@ unit and units are considered naively in the following way:
  * ``1 minutes == 60 seconds``
  * ``1 hours == 60 minutes``
  * ``1 days == 24 hours``
- * ``1 weeks = 7 days``
- * ``1 years = 365 days``
+ * ``1 weeks == 7 days``
```
Comment (axic) [inline docs/units-and-global-variables.rst:25]: Could you change these and all the above to singular on the left? e.g. `1 week == 7 days`?

Plural for 1 seems to be bad 😉

Comment (chriseth) [reply docs/units-and-global-variables.rst:25]: @axic this is how it is in Solidity, I think the change is fine like this.

**Review by chriseth — APPROVED** [review]

File: docs/units-and-global-variables.rst (hunk 1 above)
Comment (axic) [reply docs/units-and-global-variables.rst:25]: It is and it is broken english :)

Comment (chriseth) [reply docs/units-and-global-variables.rst:25]: It is one way to turn a spoken language with all its complicated rules into a formal language with less rules ;-)

**Pull request merged**